### PR TITLE
Feature/3127/swagger date-time string to long

### PR DESCRIPTION
**Pull request by garyluu**
Convert date-time string to long

Fixes dockstore/dockstore#3127

Merging this PR will auto-close the issue.  There is not really any user-facing impact which would've made it odd as a review ticket.  Just code-review and make sure there's no `date-time` at all mentioned in the swagger.yaml.

### CODE REVIEW

File: dockstore-webservice/src/main/java/io/dockstore/webservice/core/CollectionEntry.java [hunk 1]
```diff
@@ -4,12 +4,14 @@
 import java.util.Date;
 
 import io.dockstore.common.SourceControl;
+import io.swagger.annotations.ApiModelProperty;
```
Comment (denis-yuen) [inline dockstore-webservice/src/main/java/io/dockstore/webservice/core/CollectionEntry.java:7]: Question: what happens in the openapi models right now? Are they auto-detected as long or as date-time strings?

Comment (denis-yuen) [reply dockstore-webservice/src/main/java/io/dockstore/webservice/core/CollectionEntry.java:7]: Also, surprised there is no swagger.yaml file update

Comment (garyluu) [reply dockstore-webservice/src/main/java/io/dockstore/webservice/core/CollectionEntry.java:7]: the openapi models are date-time strings (not good), but we don't really use any openapi annotations.

as for the swagger.yaml file, delaying as long as possible to push due to merge issues

**Comment by coverbeck** [comment]
Whoops, only saw Denis' comment after I put mine in.

**Comment by garyluu** [comment]
> For openapi, it says it's a string
> 
> ```
>         dbCreateDate:
>           format: date-time
>           type: string
> ```
> 
> That is wrong too. I know it's not in the original issue, but could you either fix that too or log another ticket?

https://ucsc-cgl.atlassian.net/browse/SEAB-1584 since the changes are literally longer than the original issue

**Pull request merged**